code-challenges.test.js

```diff
@@ -17,43 +17,79 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe ( 'When "divisibleByThree" is called', () => {
+    it ( "return '15' is divisible by three", () => {
+        const numberOne = 15
+        const thisWorked = divisibleByThree(numberOne);
+        expect (thisWorked).toEqual(`${numberOne} is divisible by three`)
+    })
+    it ( "return '0' is divisible by three", () => {
+        const numberTwo = 0
+        const thisWorkedAgain = divisibleByThree(numberTwo);
+        expect (thisWorkedAgain).toEqual(`${numberTwo} is divisible by three`)
+    })
+    it ( "return '-7' is not divisible by three", () => {
+        const numberThree = -7
+        const wowThirdTime = divisibleByThree(numberThree);
+        expect (wowThirdTime).toEqual(`${numberThree} is not divisible by three`)
+    })
+})
+
+// // b) Create the function that makes the test pass.
+
 var num1 = 15
 // Expected output: "15 is divisible by three"
-
 var num2 = 0
-// Expected output: "0 is divisible by three"
-
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
+const divisibleByThree = (testedValue) => {
+    if (testedValue % 3 ===0){
+        return `${testedValue} is divisible by three`
+    } 
+    else if (testedValue % 3 !==0){
+        return `${testedValue} is not divisible by three`
+    }
+}
 
 
-
-
-
-// b) Create the function that makes the test pass.
-
-
-
-
-
+ 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
 var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 
 var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
-
-
-
-
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+
+describe ( 'When "capitalizeFirstLetter" is called', () => {
+    it ("return 'arrayCapitalized'", () => {
```

Comment:
generally for `it` test description we want to keep with plain English. Outside of using the method name in the `describe` description, we should be describing the outcomes with words that do not look like code or reference methods in code